phoenix-core/src/main/java/org/apache/phoenix/schema/types/PhoenixArray.java

```diff
@@ -561,6 +666,23 @@ public int estimateByteSize(int pos) {
 		}
 		
 		@Override
+		public String toString() {
+			StringBuilder sb = new StringBuilder(TO_STRING_BEGIN);
+			boolean isFirst = true;
+			final Format defaultFormat = null;
+			for (Byte o : byteArr) {
+				if (isFirst) {
+					isFirst = false;
+				} else {
+					sb.append(TO_STRING_SEPARATOR);
+				}
+				sb.append(this.baseType.toStringLiteral(o, defaultFormat));
```

Comment:
Also it looks like toStringLiteral() is taking Object. So is it possible to just use the toString() method defined in the super class PhoenixArray. This way you won't have to duplicate code for toString() method for every array type.